components/scream/tests/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+file(GLOB SCREAM_TEST_SRCS "*.cpp")
```

Comment:
So far is innocuous, and probably will always be. However, I'm kind of against "globbing" files with cmake. I rather see a list of the files. I find it safer and clearer.

Reply:
I hate the thought of having to modify CMake every time a file is added.

Reply:
Modifying a CMakeLists.txt will trigger cmake to re-configure if one pulls a branch where someone just added a cpp file. With GLOB, cmake won't see any change, and Makefile's won't be updated. Besides, globbing makes slightly harder to git-bisect code (again, cause cmake targets may not change).

Not major issues, granted. But I like to avoid implicit behaviors in favor of more self-explaining code.

But that's my personal opinion. I can handle either way.

Reply:
Yeah, that's a bit of a downside. I've always very frequently re-configured my build areas (re-run cmake), so I prefer the glob approach over having to modify CMakeLists for every new source file.

Reply:
I'm also on the explicit list side of things, but don't have a strong preference.

Reply:
Adding a new file doesn't happen very often once a project matures.  Explicit is better.